tests/cypress/support/helpers/tenderly/vnet.ts

```diff
@@ -76,17 +114,71 @@ export function withVirtualTestnet(opts: (uuid: number) => DeepPartial<CreateVir
 
     const finalOpts = Cypress._.merge({}, defaultOpts, opts(uuid))
 
-    createVirtualTestnet({ ...tenderlyAccount, ...finalOpts }).then((created) => (vnet = created))
+    createVirtualTestnetRequest({ ...tenderlyAccount, ...finalOpts }).then((created) => (vnet = created))
+  })
+
+  after(() => {
+    if (!vnet) return
+
+    const deleteOpts: DeleteVirtualTestnetOptions = {
+      vnetId: vnet.id,
+    }
+
+    deleteVirtualTestnetRequest({ ...tenderlyAccount!, ...deleteOpts })
```

Comment:
if deleteVirtualTestnetRequest is properly typed, DeleteVirtualTestnetOptions should be inferred, right?
```suggestion
    deleteVirtualTestnetRequest({ ...tenderlyAccount!, vnetId: vnet.id })
```

Reply:
Ah yeah, kept the explicit typing out of consistency, but can remove